### ISSUE TITLE
Complete Undo Close Tab is not working

### ISSUE THREAD

**Issue by ofdeath** [smoothgesturesplus/Issues#68]
I have installed SmoothGesturesPlus and SmoothGesturesPlusExtras on OSX Mojave.
Right click is working fine - I think it means that Extras is working fine.
But Undo Close Tab opens the last closed tab without history.
I expect Undo Close Tab opens the last closed tab with history like Cmd+Shift+T does if I installed SmoothGesturesPlugExtras.
Did I do something wrong? Or it's just not possible?

----
Undo Close Tab
Open the last closed tab. This only loads the URL of the page; For a more complete version install the Extras

**Comment by smoothgesturesplus** [comment]
Yes, if you are running the Extras on macOS, then the full functionality of Undo Close Tab should be available. I'll take a closer look. In the meantime, can you restart your computer and confirm that you still see the issue? (Right click works, Undo Close Tab doesn't)

**Comment by ofdeath** [comment]
I have restarted my computer but it's still not working even though right click is working - one right click opens context menu. 
I have tried to use new extension "smartup Gestures" and Reopen Closed Tab is working! Although right click is not working.
Anyway it requires a permission of session management(?) to activate Reopen Closed Tab, does this problem be related with a permission of session management?
